play/src/i18n/ca-ES/camera.ts

```diff
@@ -19,6 +19,15 @@ const camera: DeepPartial<Translation["camera"]> = {
             chrome: "/resources/help-setting-camera-permission/en-US-firefox.png",
         },
     },
+    webrtc: {
+        title: "WebRtc connection error",
+        error: "STUN / TURN server isn't reachable",
+        content:
+            "If you are connecting via a VPN, please try to logout and refresh the web page. You may click on the link below to test your WebRtc connection.",
+        testUrl: "WebRtc connection test",
+        refresh: "Refresh",
+        continue: "Continue",
+    },
```

Comment:
I wonder if this is compulsory for all languages. I think you can keep this empty instead.

Reply:
I think that we should improve the message and UI/UX interaction with the user if there are any errors. So I guess it's important for us to have specific instruction with all languages?